### DISCUSSION TITLE
Installing Audiobookshelf on Linux Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by Hallo951** [advplyr/audiobookshelf#1535] [Q&A]
Hi there,
Is there anywhere a tutorial on how to install audiobookshelf on linux ubuntu 22.04? Currently I have audiobookshelf running in docker on my synology NAS, but I find it very sluggish when editing metadata. I would like to try it on a more powerful Linux-based machine, but I am a Linux novice.

Thanks

**Comment by susan3333333** [comment]
I don't know about Ubuntu specifically, but I used the Docker install instructions on my OpenSuse Tumbleweed pc with no problem, and I know nothing about Docker but mine is running fine in spite of me :)

**Comment by advplyr** [comment]
The only other install method is the PPA and there are instructions on using that in the docs. I don't think that will improve your performance though. I'm not sure what you mean by sluggish but there could be many reasons for bad performance that has to do with your setup.
